packages/gestalt/src/SelectList.jsdom.test.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React from 'react';
+import { render } from '@testing-library/react';
+import SelectList from './SelectList.js';
+
+const options = [
+  { label: 'option1', value: 'value1' },
+  { label: 'option2', value: 'value2' },
+  { label: 'option3', value: 'value3' },
+];
+
+it('Renders an FormErrorMessage if an error message is passed in', () => {
+  const { getByText } = render(
+    <SelectList
+      errorMessage="Error message"
+      id="test"
+      onChange={jest.fn()}
+      options={options}
+    />
+  );
+
+  expect(getByText('Error message')).toBeVisible();
+});
+
+it('SelectList with name', () => {
+  const { container } = render(
+    <SelectList
+      name="select_name"
+      id="select_id"
+      onChange={jest.fn()}
+      options={options}
+    />
+  );
+  expect(container.querySelector('[name="select_name"]')).toBeVisible();
+});
+
+it('SelectList with errorMessage prop change', () => {
+  const handleChange = jest.fn();
+  const { getByText, rerender } = render(
+    <SelectList id="test" onChange={handleChange} options={options} />
+  );
+  expect(() => {
+    getByText('Error message');
+  }).toThrow('Unable to find an element with the text: Error message');
+  rerender(
+    <SelectList
+      id="test"
+      onChange={handleChange}
+      options={options}
+      errorMessage="Error message"
+    />
+  );
+  expect(getByText('Error message')).toBeVisible();
+});
+
+it('SelectList with disabled', () => {
+  const { container } = render(
+    <SelectList disabled id="test" onChange={jest.fn()} options={options} />
+  );
+  expect(container.querySelector('select[disabled]')).toBeVisible();
```

Comment:
I wonder if we should just invest in building a few more selectors into RTL. This one seems legit, but annoying that we need to use the querySelector hack.

Reply:
Agreed - seems like a good follow up. This initial PR is to remove enzyme and we can improve our tests later.